src/fold/fold.rs

```diff
@@ -153,7 +153,7 @@ fn fold_file<T: Read>(mut file: BufReader<T>, bytes: bool, spaces: bool, width:
                                 match rfind_whitespace(slice) {
                                     Some(m) => {
                                         let routput = &slice[m + 1 .. slice.chars().count()];
-                                        let ncount = routput.chars().fold(0usize, |out, ch: char| {
+                                        let ncount = routput.chars().fold(0, |out, ch: char| {
```

Comment:
removes a warning about the block returning -1 for an unsigned, would appreciate if someone could tell me if this can cause a bug